frontend/src/app/organizations/organizations-list/organizations-list.component.html

```diff
@@ -0,0 +1,65 @@
+<div fxLayout="column" fxLayoutGap="100px">
+    <mat-table #table [dataSource]="organizationMock">
+        <ng-container matColumnDef="Name">
+            <mat-header-cell *matHeaderCellDef> Name </mat-header-cell>
+            <mat-cell *matCellDef="let element"> {{element.name}} </mat-cell>
+        </ng-container>
+        <ng-container matColumnDef="Beschreibung">
+            <mat-header-cell *matHeaderCellDef> Beschreibung </mat-header-cell>
+            <mat-cell *matCellDef="let element"> {{element.description}} </mat-cell>
+        </ng-container>
+        <ng-container matColumnDef="Strasse">
+            <mat-header-cell *matHeaderCellDef> Strasse </mat-header-cell>
+            <mat-cell *matCellDef="let element"> {{element.address.street}} </mat-cell>
+        </ng-container>
+        <ng-container matColumnDef="Hausnummer">
+            <mat-header-cell *matHeaderCellDef> Hausnummer </mat-header-cell>
+            <mat-cell *matCellDef="let element"> {{element.address.streetNumber}} </mat-cell>
+        </ng-container>
+        <ng-container matColumnDef="PLZ">
+            <mat-header-cell *matHeaderCellDef> PLZ </mat-header-cell>
+            <mat-cell *matCellDef="let element"> {{element.address.zipcode}} </mat-cell>
+        </ng-container>
+        <ng-container matColumnDef="Stadt">
+            <mat-header-cell *matHeaderCellDef> Stadt </mat-header-cell>
+            <mat-cell *matCellDef="let element"> {{element.address.city}} </mat-cell>
+        </ng-container>
+        <ng-container matColumnDef="Telefonnummer">
+            <mat-header-cell *matHeaderCellDef> Telefonnummer </mat-header-cell>
+            <mat-cell *matCellDef="let element"> {{element.contact.phoneNumber}} </mat-cell>
+        </ng-container>
+        <ng-container matColumnDef="E-Mail">
+            <mat-header-cell *matHeaderCellDef> E-Mail </mat-header-cell>
+            <mat-cell *matCellDef="let element"> {{element.contact.mail}} </mat-cell>
+        </ng-container>
+        <ng-container matColumnDef="Faxnummer">
+            <mat-header-cell *matHeaderCellDef> Faxnummer </mat-header-cell>
+            <mat-cell *matCellDef="let element"> {{element.contact.fax}} </mat-cell>
+        </ng-container> -->
+        <ng-container matColumnDef="Zugehörige">
+            <mat-header-cell *matHeaderCellDef> Zugehörige </mat-header-cell>
+            <mat-cell *matCellDef="let element"> 
+                <mat-form-field>
+                    <mat-label>Zugehörige</mat-label>
+                    <mat-select [(ngModel)]="element.employees">
+                        <mat-option *ngFor="let employee of element.employees" [value]="employee">
+                            {{employee.firstName + ' ' + employee.lastName}}
+                        </mat-option>
+                    </mat-select>
+                </mat-form-field>
+            </mat-cell>
+        </ng-container>
```

Comment:
Muss nochmal mit Pana abgesprochen werde glaub ich. Ich würde diese Information in die Detail ansicht packen. Vermutlich auch Email, Faxnummer und Telefonnummer ,...

Reply:
jap offensichtlich..na hoffentlich habe ich mich nicht ganz umsonst mit dem drop down menu beschäftigt :(